app/main.py

```diff
@@ -0,0 +1,92 @@
+import sys
+from datetime import datetime, timezone
+
+from config.logging import setup_logging
+from config.settings import settings
+from google_calendar.google_calendar_api import GoogleCalendarService
+from scrapers.football_scraper import FootballScraper
+
+logger = setup_logging(__name__)
+
+
+def update_calendar(calendar_service: GoogleCalendarService):
+    """
+    Update the calendar with upcoming matches from the Maccabi TLV site.
+    """
+    # Current DateTime - Update and add upcoming matches only
+    current_time = datetime.now(timezone.utc).replace(tzinfo=None).isoformat() + "Z"
+    logger.info("Current time:")
+    logger.info(current_time)
+
+    if settings.DELETE_ALL_MATCHES:
```

Comment:
That a bit weird to have this code under update, maybe we can extract this to be one function above and choose the right action we should do? like "id delete: delete", if not, if update: update()" ...

Reply:
Moved to main function.
In some cases we might want to delete all events before inserting updated\new event